scripts/scripts.js

```diff
@@ -226,5 +274,8 @@ const { loadArea, setConfig, getMetadata } = await import(`${miloLibs}/utils/uti
   await buildAutoBlocks();
   overrideMiloBlocks();
   await loadArea();
+  await decorateArticlePageUI();
+  decorateMediaBlock();
+  decorateMasonryBrick();
```

Comment:
@snowiewdev I think we'll want to move these functions into a different file where they're only used as needed. Also, I'm not sure but the decorateMediaBlock and masonryBrick sound a lot like features in milo already that we may be able to just use if the structure is the same. `MILO_TEMPLATES` might be the place for these.